channel/state/outcome/allocation.go

```diff
@@ -18,10 +18,10 @@ const (
 
 // Allocation declares an Amount to be paid to a Destination.
 type Allocation struct {
-	Destination    types.Destination // Either an ethereum address or an application-specific identifier
-	Amount         *types.Uint256    // An amount of a particular asset
-	AllocationType AllocationType    // Directs calling code on how to interpret the allocation
-	Metadata       []byte            // Custom metadata (optional field, can be zero bytes). This can be used flexibly by different protocols.
+	Destination    types.Destination `json:"destination"`     // Either an ethereum address or an application-specific identifier
```

Comment:
can't we remove those json instructions?

Reply:
I can actually just on deserialization in map we will have values with exactly the same name as here